src/test/java/com/google/step/datamanager/DealManagerDatastoreTest.java

```diff
@@ -206,6 +207,116 @@ public void testUpdate_allFields() {
     assertEquals(expected, updatedDeal);
   }
 
+  @Test
+  public void testGetDealPublishedByUsers_success() {
+    // Add deals published by USER_ID_A and USER_ID_B
+    Deal dealA =
+        dealManagerDatastore.createDeal(
+            DESCRIPTION_A,
+            BLOBKEY_A,
+            DATE_A,
+            DATE_B,
+            SOURCE_A,
+            USER_ID_A,
+            RESTAURANT_ID_A,
+            EMPTY_LIST);
+    Deal dealB =
+        dealManagerDatastore.createDeal(
+            DESCRIPTION_A,
+            BLOBKEY_A,
+            DATE_A,
+            DATE_B,
+            SOURCE_A,
+            USER_ID_B,
+            RESTAURANT_ID_A,
+            EMPTY_LIST);
+    Deal dealC =
+        dealManagerDatastore.createDeal(
+            DESCRIPTION_A,
+            BLOBKEY_A,
+            DATE_A,
+            DATE_B,
+            SOURCE_A,
+            USER_ID_C,
+            RESTAURANT_ID_A,
+            EMPTY_LIST);
+
+    // Get deals published by the users followed by USER_ID_A
+    List<Deal> dealsForA =
+        dealManagerDatastore.getDealsPublishedByUsers(
+            new HashSet<Long>(Arrays.asList(USER_ID_B, USER_ID_C)));
+    assertEquals(2, dealsForA.size());
```

Comment:
```suggestion
    assertEquals(dealsForA.size(), 2);
```

to be consistent with your other assertions

Reply:
i fixed the other assertions instead since the first argument should be the expected value, and to be consistent with the other tests